### ISSUE TITLE
can you explain the noise loss function?

### ISSUE THREAD

**Issue by Zruto** [aaaaangel/RRDNet#4]
> def noise_loss(image, illumination, reflectance, noise):
    weight_illu = illumination
    weight_illu.detach()
    loss = weight_illu*noise
    return torch.norm(loss, 2)
